recipes/diffpy.snmf/meta.yaml

```
@@ -0,0 +1,62 @@
+{%- set name = "diffpy.snmf" -%}
+{%- set version = "0.1.0" -%}
+{%- set sha256 = "2903bd7d9df588b35bd45bc07ea745c69fb281fb774beef8af645cebcb980b7c" -%}
```

Please set the sha256 explicitly
```suggestion
{%- set version = "0.1.0" -%}
```